mteb/tasks/Clustering/MasakhaNEWSClusteringP2P.py

```diff
@@ -0,0 +1,58 @@
+import datasets
+import numpy as np
+
+from ...abstasks.AbsTaskClustering import AbsTaskClustering
+
+
+class MasakhaNEWSClusteringP2P(AbsTaskClustering):
+    @property
+    def description(self):
+        return {
+            "name": "MasakhaNEWSClusteringP2P",
+            "hf_hub_name": "masakhane/masakhanews",
+            "description": (
+                "Clustering of news article headlines and texts from MasakhaNEWS dataset. Clustering of 10 sets on the news article label."
+            ),
+            "reference": "https://huggingface.co/datasets/masakhane/masakhanews",
+            "type": "Clustering",
+            "category": "p2p",
+            "eval_splits": ["test"],
+            "eval_langs": ["fr"],
```

Comment:
Adding other languages in this Clustering task requires to inherit from `MultilingualTask`, and modify the `evaluate()` function in `AbsTaskClustering` by adding a condition to check if the task is_multilingual (for the moment all clustering tasks are monolingual). This to load the dataset for the evaluated language (just as it was done in `AbsTaskClassification`). 
Note that this will also change the json file where the results will be saved, there will be a new key 'lang' before each result.
We can make the changes if you're okay with it ?

Reply:
I see, I think we'd have to do it one way or another at some point, e.g. in this PR it was added for Reranking: https://github.com/embeddings-benchmark/mteb/pull/227/files

So I think it would be worth it, but of course up to you. For the jsons, I think it just needs a short script to load them and then place the results in a `fr` sub dictionary